packages/itinerary-body/src/defaults/route-description-footer.tsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/prefer-default-export */
+import React, { ReactElement } from "react";
+import * as S from "../styled";
+import { RouteDescriptionFooterProps } from "../types";
+
+export function DefaultRouteDescriptionFooter({
+  onClick = null,
+  waitMinutes
+}: RouteDescriptionFooterProps): ReactElement {
+  const JSX =
+    waitMinutes > 0 ? (
+      <S.ArrivalTimeContainer onClick={onClick}>
+        Arrives in {waitMinutes} minutes
+      </S.ArrivalTimeContainer>
+    ) : null;
+
+  return JSX;
```

Comment:
```suggestion
  return waitMinutes > 0 ? (
      <S.ArrivalTimeContainer onClick={onClick}>
        Arrives in {waitMinutes} minutes
      </S.ArrivalTimeContainer>
    ) : null;
```

This can be inlined!

Reply:
Done.